### PR TITLE
Fix the tween null check

### LINKED ISSUE #45399: Project crashes with no errors upon tweening a non existent node
**Godot version:**
3.2.3 stable

**OS/device including version:**
MacOS 11.0.1 Big Sur, GLES3

**Issue description:**
I created a tween inside of a script, added it to the scene tree, and tweened a non-existing node. Instead of leaving an error message, the game crashes without a trace in the debugger. 

**Steps to reproduce:**
Create an empty scene, add a root node and add a script to this node, in the script create a new tween and add it to the scene tree. Then you only need to make the tween interpolate a property of a non-existent node and run the project, no need to even call the start() method on the tween. The project immediately crashes when interpolating the property.

**Minimal reproduction project:**
[Bug.zip](https://github.com/godotengine/godot/files/5861051/Bug.zip)

### PR DESCRIPTION
**Pull request by Norrox**
well this fixes the null check for tween for 4.0, the 3.2 code looks a little different so i dont know how to push both to the same master as this is my first time playing around with this :)

fixes #45399 error for version 4

### CODE REVIEW

**Comment by Norrox** [comment]
@Calinou is this ok? ;D

**Comment by Calinou** [comment]
> @Calinou is this ok? ;D

I don't merge code pull requests on the main repository; you'll have to wait for Akien to take a look.

File: scene/animation/tween.cpp [hunk 1]
```diff
@@ -1350,6 +1350,9 @@ void Tween::interpolate_property(Object *p_object, NodePath p_property, Variant
 		return;
 	}
 
+	// Check that the target object is valid
+	ERR_FAIL_COND(p_object == nullptr);
```
Comment (Calinou) [inline scene/animation/tween.cpp:1354]:
```suggestion
	ERR_FAIL_COND_MSG(p_object == nullptr, "The node to tween is null. Was it removed before tweening?");
```

File: scene/animation/tween.cpp [hunk 2]
```diff
@@ -1378,6 +1381,9 @@ void Tween::interpolate_method(Object *p_object, StringName p_method, Variant p_
...
```
Comment (Calinou) [inline scene/animation/tween.cpp:1385]:
```suggestion
	ERR_FAIL_COND_MSG(p_object == nullptr, "The node to tween is null. Was it removed before tweening?");
```

**Review by Calinou — DISMISSED** [review]
I think it'd be good to add a descriptive message to help the user troubleshoot the issue:

**Comment by akien-mga** [comment]
I force pushed a rebase to squash the commits into one and clarify the commit message.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**